<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer::Internal::MerHardwareDeviceWizardSetupPage</class>
 <widget class="QWizardPage" name="Mer::Internal::MerHardwareDeviceWizardSetupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>247</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Configuration name:</string>
     </property>
     <property name="buddy">
      <cstring>configLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="configLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="hostLabel">
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="hostLabelEdit">
     <property name="text">
      <string>none</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="userNameLabel">
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="usernameLabelEdit">
     <property name="text">
      <string>none</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="sshPortLabel">
     <property name="text">
      <string>SSH Port:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="sshPortLabelEdit">
     <property name="text">
      <string>none</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="freePortsLabel">
     <property name="text">
      <string>Free ports:</string>
     </property>
     <property name="buddy">
      <cstring>freePortsLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="freePortsLineEdit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="freePortsDebuggerMessageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Note: For debugging, at least 2 free ports are required.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
